main/zoe/guide/contracts/autoswap.md

```diff
@@ -2,161 +2,170 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/autoswap.js) (Last updated: 4/22/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/2a8b0fc2ece7344604bcc23b295367cd871f6995/packages/zoe/src/contracts/autoswap.js) (Last updated: 2020-9-14)
 ##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
 
-::: tip Out-of-date status
-Zoe's master branch is currently an Alpha release candidate. This doc
-and its underlying contract are in the process of being updated, and should be current with the release candidate in another few days. What you see here is out of date. We apologize for any inconvenience this may cause.
-:::
+Autoswap is a contract that maintains a pool of assets (the 'liquidity pool') that can
+always trade against any incoming offer. It can do this because it uses a rule to set
+the prices, so the price is updated as its assets change.  This contract uses the
+[constant-product](https://github.com/runtimeverification/verified-smart-contracts/blob/uniswap/uniswap/x-y-k.pdf)
+rule.
+
+This contract follows the design of [UniSwap](https://uniswap.org/), with a single
+pool. This means any installation of autoswap can make exchanges between two
+issuers. Our multipoolAutoswap generalizes this to many pools, all of which share a
+common intermediate pool. We leave this single autoswap contract available because it's
```

Comment:
"We make" ("leave" makes this sound like we're doing the dregs a favor)

Reply:
done